components/monitoring/logging/staging/configure-logging/cluster-log-forwarder-health-check.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cm
+  namespace: openshift-gitops
```

Comment:
This won't work since we are deploying using an operator. https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#resource-customizations
Maybe it would be better to contribute it to the upstream project https://github.com/argoproj/argo-cd/tree/master/resource_customizations

I think that this check should be generic and not check a specific log forwarder (e.g splunk/cloudwatch).